modules/analyze.py

```diff
@@ -0,0 +1,62 @@
+import json
```

Comment:
I recommend including this script as an executable in the `bin` directory and calling it from there. That is considered Nextflow best practice. An example of that pattern can be seen in the nf-synapse repo https://github.com/Sage-Bionetworks-Workflows/nf-synapse